docs/docs/config.md

```diff
@@ -138,6 +138,7 @@ client_id                      | STRING                              | unique te
 replica_server_id              | LONG                                | unique numeric identifier for this maxwell instance | 6379 (see notes)
 master_recovery                | BOOLEAN                             | enable experimental master recovery code            | false
 gtid_mode                      | BOOLEAN                             | enable GTID-based replication                       | false
+ignore_producer_error          | BOOLEAN                             | terminate Maxwell on Kafka/Kinesis errors if false  | true
```

Comment:
Describing the `false` case is a bit confusing. Perhaps "log producer errors instead of terminating maxwell"?

Alternatively, we could do something more explicit like `producer_error_action=log|terminate`, but it might not be worth the effort - we probably won't ever have a third action.